setup/raspberry-pi/ansible-playbook.yaml

```diff
@@ -50,7 +55,6 @@
 
     - name: User | Download pubkey(s)
       get_url: url=https://github.com/{{ item }}.keys dest=/tmp/{{ item }}.keys
-      delegate_to: 127.0.0.1
```

Comment:
I've also had a problem with this, because sudo is yes and it tries to sudo in the machine that's running ansible. I've changed to
```
      delegate_to: 127.0.0.1
      become: no
```
and it's ok. Well, either way should be good as long as the lookup below works.